lib/commands/compatibility.js

```diff
@@ -2,7 +2,7 @@
 
 const Chalk = require('chalk');
 const Program = require('commander');
-const semver = require('semver');
+const { compareVersions, compareVersionsWithComperator } = require('composer-version-constraint-evaluator');
```

Comment:
AFAIK the correct English word is "comparator".
```suggestion
const { compareVersions, compareVersionsWithComparator } = require('composer-version-constraint-evaluator');
```

Reply:
Oh damn. Yes, you're right